index.js

```diff
@@ -163,6 +163,7 @@ module.exports = (cmd, args, opts) => {
 
 	const processDone = new Promise(resolve => {
 		spawned.on('exit', (code, signal) => {
+			spawned.killed = signal !== null;
```

Comment:
If this turns out to be true, we may want to reconsider the whole idea. With the current implementation, at least `cp.killed` is consistent across platforms. Maybe we just attach `signal`.

I worry magically doing this for the user on only one platform will create confusing bugs.